app/serializers/full_auth_token_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class FullAuthTokenSerializer < AuthTokenSerializer
```

Comment:
I find this naming a bit odd - it would be easier to understand if this was named `AuthToken...Serializer`